src/main/java/com/elixr/poc/bulkipmort/service/FileUploadService.java

```diff
@@ -0,0 +1,24 @@
+package com.elixr.poc.bulkipmort.service;
+
+import com.elixr.poc.common.MessagesKeyEnum;
+import com.elixr.poc.common.exception.ExtensionException;
+import com.elixr.poc.common.util.MessagesUtil;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+@Service
+public class FileUploadService {
+
+    /**
+     * Accepting only .csv files.
+     *
+     * @param file
+     * @return
+     */
+    public boolean uploadFile(MultipartFile file) {
+        if (!file.getOriginalFilename().endsWith(".csv")) {
+            throw new ExtensionException(MessagesUtil.getMessage(MessagesKeyEnum.ENTITY_FILE_EXTENSION.getKey()));
+        }
+        return true;
```

Comment:
I am not questioning the logic here but I see you are simply returning true without doing any upload logic. If this is the requirement given to you then you can ignore this comment but if you feel that you have missed to write the file upload logic to some location in the laptop, then you can do it here.

Reply:
Yes, the logic part is still in Progress.
I'm trying to read the file and so on

Reply:
Ok. then I will approve after you complete and I finish reviewing that logic.